src/_articles/archive/zones.md

```diff
@@ -193,9 +207,9 @@ they still execute.
 As a consequence a zoned error handler might
 be invoked multiple times.
 
-Also note that an _error zone_—a zone that has an error handler—might
-handle errors that originate in a child (or other descendant)
-of that zone.
+Also note that an _error zone_ — a zone that has an error handler
+— might handle errors that originate in a child
+(or other descendant) of that zone.
 A simple rule determines where
 errors are handled in a sequence of future transformations
 (using `then()` or `catchError()`):
```

Comment:
**suggestion:** Rephrase to tie this back to the last sentence. Also, the callout is the rule, so you don't need to preface it with a sentence, then a callout. It's too much. 

```markdown
A zoned error handler exists in an _error zone_.
This error zone might handle errors that originate
in a descendant of that zone.
```

Reply:
**suggestion:** Maybe include the sentence after the note within the note. That would tie the thoughts together better.